NDB.Covid19/NDB.Covid19/WebServices/ImportantMessageService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading.Tasks;
+using NDB.Covid19.Configuration;
+using NDB.Covid19.Models;
+using NDB.Covid19.PersistedData;
+
+namespace NDB.Covid19.WebServices
+{
+    public class ImportantMessageService : BaseWebService
+    {
+        public async Task<ImportantMessage> GetImportantMessage()
+        {
+            ApiResponse<ImportantMessage> response = await Get<ImportantMessage>($"{Conf.URL_GET_IMPORTANT_MESSAGE}?lang={LocalPreferencesHelper.GetAppLanguage()}");
```

Comment:
I assume this will be changed? Shall we wait until @NC-001 gets her PR in first?

Otherwise, all looks goods.

Reply:
Discussed this with @NC-001 and agreed that this should be merged.